src/components/Settings/Rounding.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { Header } from 'semantic-ui-react';
+
+import { setRounding, setRoundingDown } from '../../actions/settings';
+import { valueFromEventTarget } from '../../core/dom';
+import Combo from './Combo';
+
+type RoundingProps = {
+  rounding: string;
+  roundingDown: string;
+  setRoundingProp: (value: any) => void;
```

Comment:
I would rather name this `onSetRounding` or just `setRounding`.

Value is of type `string` if I am not mistaken.

Reply:
I think it needs to be onSetRounding as the setRounding is reserved for the action setRounding.